tooling/bundler/src/bundle/windows/nsis.rs

```diff
@@ -71,7 +71,6 @@ pub fn bundle_project(settings: &Settings, updater: bool) -> crate::Result<Vec<P
   let nsis_toolset_path = tauri_tools_path.join("NSIS");
 
   if !nsis_toolset_path.exists() {
-    create_dir_all(&nsis_toolset_path)?;
```

Comment:
at least we should create the parent of `nsis_toolset_path` so instead of removing, we should move this call up to line 70 and call it for `tauri_tools_path`

Reply:
Okay, I get it.